members.yaml

```diff
@@ -195,9 +195,99 @@ paths:
             application/json:
               schema:
                 $ref: 'https://raw.githubusercontent.com/wwi16ama/api_spec/master/openapi.yaml#/components/schemas/Error'
+  '/members/{id}/changePassword':
```

Comment:
Since you approved your own PR by yourself Im rewriting my suggestions again
Small suggestion. How about '/members/{id}/changePasswordAsMember' for a more descriptive route name